src/test/java/com/google/step/snippet/W3SchoolClientTest.java

```diff
@@ -0,0 +1,60 @@
+package com.google.step.snippet;
+
+import com.google.step.snippet.data.Card;
+import com.google.step.snippet.external.W3SchoolClient;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class W3SchoolClientTest {
+
+  W3SchoolClient client = new W3SchoolClient();
+
+  @Test
+  public void htmlImage() {
+    Card actual = client.search("https://www.w3schools.com/tags/tag_img.asp");
+    Card expected =
+        new Card(
+            "HTML <img> Tag",
+            "<img src=\"img_girl.jpg\" alt=\"Girl in a jacket\" width=\"500\" height=\"600\">",
+            "https://www.w3schools.com/tags/tag_img.asp",
+            "How to insert an image:");
+    Assert.assertTrue(actual.equals(expected));
```

Comment:
Use assertEquals. Always go for the most semantic assert so that failing test errors are more descriptive e.g. it'll say "foo is not equal to bar as expected" instead of "this boolean is not true as expected"